guides/05_Simulcast.md

```diff
@@ -18,7 +18,15 @@ Simulcast support is built into the WebRTC library in the majority of browsers.
 
 The recommended resolution in JS SDK is 1280x720. This resolution will provide the most flexibility and allow subscribers to maintain smoother video quality transitions.  In certain circumstances, like mobile browsers, the input resolution may not be high enough to support all desired simulcast streams, so the logic will adapt appropriately to send as many as possible.
 
-The SimulcastUplinkPolicy configures RTCRtpSender to have three encoding layers, but only ever enables two of them.  Which two are enabled is based on a variety of factors.   Experiments show that configuring RTCRtpSender with three encoding layers up front and dynamically enabling and disabling layers provides a better experience and reduces the burden of having to dynamically  manage capture resolution and encoding parameters.
+The SimulcastUplinkPolicy configures RTCRtpSender to have three encoding layers, but only ever enables two of them.  Which two are enabled is based on a variety of factors. Experiments show that configuring RTCRtpSender with three encoding layers up front and dynamically enabling and disabling layers provides a better experience and reduces the burden of having to dynamically  manage capture resolution and encoding parameters.
+
+### Browser compatibility
+
+|Browser              |Supported versions  |
+|---                  |---                 |
+|Chrome               |76+                 |
+
+> Note: Simulcast in Amazon Chime SDK for JavaScript should also work in Chromium-based browsers on desktop like Chromium Edge and Opera; and on mobile like Chromium-Webview and Samsung Internet.
```

Comment:
Is it possible to get the actual version numbers of those browsers at the M76 point (maybe we can estimate too).

Reply:
Our documentation should not routinely say "should work". Either we've tested it and it works (or doesn't), or it should be an uncommon case that most customers won't care about. I would argue that Edge is in the first set, not the second.

Reply:
Agree, I checked with internal QA around this and tested few but "should work" is not ideal, and will need more time to actually get all those browsers tested as mentioned here with the list supported version thanks! I have created an internal ticket to track this work and add support for Simulcast like we have in VF guide.